build_tools/docker/bazel/Dockerfile

```diff
@@ -19,7 +19,8 @@
 
 # Run interactively using the following, where IREE_WORKDIR is the path to your
 # local dev environment:
-# docker run -it -rm --entrypoint bash --volume ${IREE_WORKDIR}:/usr/src/iree/ \
+# docker run -it -rm --entrypoint bash \
+#   --volume "${IREE_WORKDIR}:/usr/src/iree/" \
```

Comment:
Add a `?` while you're here? Note this is not required by Google style, but it's a convention I started because I hate no error message for undefined variables and have been bitten by it before so now it's consistent with other places :-P Even if you literally just defined it, typos are not uncommon (like I actually did that last week).

```suggestion
#   --volume "${IREE_WORKDIR?}:/usr/src/iree/" \
```

Reply:
Done.